Harmony/Harmony.csproj

```diff
@@ -77,7 +64,7 @@
 		<PackageReference Include="jnm2.ReferenceAssemblies.net35" Version="1.0.1" PrivateAssets="all" Condition="'$(TargetFramework)'=='net35'">
 			<IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
 		</PackageReference>
-		<PackageReference Include="ILRepack.MSBuild.Task" Version="2.0.13" PrivateAssets="All" />
+        <PackageReference Include="SourceLink.Copy.PdbFiles" Version="2.8.3" PrivateAssets="All" />
```

Comment:
What is the reason for replacing `ILRepack.MSBuild.Task` with `SourceLink.Copy.PdbFiles`? From what I can see they both solve very different issues...

Reply:
`ILRepack.MSBuild.Task` is repeated